Examples/src/components/AppFooter/AppFooter.tsx

```diff
@@ -54,21 +53,20 @@ export default function AppFooter() {
                 <Box mb={32}>
                     <h4>SciChart.js: Fast, Realtime, High Performance </h4>
                     <Box mt={8} className={classes.SiteLinks}>
-                        <Link
-                            className={classes.Link}
+                        <a
```

Comment:
add please title attribute to a tag, this good for SEO

Reply:
there already is this attribute in the line below, as well as on other links in the file